python/lsst/source/injection/utils/make_injection_pipeline.py

```diff
@@ -81,7 +74,8 @@ def make_injection_pipeline(
     reference_pipeline: Pipeline | str,
     injection_pipeline: Pipeline | str | None = None,
     exclude_subsets: bool = False,
-    excluded_tasks: set[str] | str = {
+    excluded_tasks: set[str]
+    | str = {
```

Comment:
This looks wrong. Which version of black is doing that?

`per-commit autoupdate` will fix things for you. The version of black in there is a year old.

Reply:
That was pre-commit, so that old version of black.